lib/Controller/ViewController.php

```diff
@@ -93,6 +94,7 @@ public function index():TemplateResponse {
 		$timezone = $this->config->getUserValue($this->userId, $this->appName, 'timezone', $defaultTimezone);
 		$slotDuration = $this->config->getUserValue($this->userId, $this->appName, 'slotDuration', $defaultSlotDuration);
 		$defaultReminder = $this->config->getUserValue($this->userId, $this->appName, 'defaultReminder', $defaultDefaultReminder);
+		$defaultCalendarIdUser = $this->config->getUserValue($this->userId, $this->appName, 'defaultCalendarId', $defaultCalendarId);
```

Comment:
```suggestion
		$defaultCalendarIdUser = $this->config->getUserValue($this->userId, 'dav', 'defaultCalendar', $defaultCalendarId);
```

This also needs a reasonable default ($defaultCalendarId) is undefined?!

Reply:
Instead of using the calendar id, use the calendar uri, then default to `CalDavBackend::PERSONAL_CALENDAR_URI`.

Reply:
The personal calendar always gets recreated at some point if there's nothing else